shared/src/main/scala/com/cibo/evilplot/plot/Histogram.scala

```diff
@@ -40,14 +40,14 @@ object Histogram {
   val defaultBinCount: Int = 20
 
   // Create binCount bins from the given data and xbounds.
-  private def createBins(values: Seq[Double], xbounds: Bounds, binCount: Int): Seq[Point] = {
+  private def createBins(values: Seq[Double], xbounds: Bounds, binCount: Int, normalize: Boolean = false): Seq[Point] = {
```

Comment:
Out of curiosity, the other createBins parameters are given defaults in the apply method, including normalize. Does adding `Boolean = false` here carry an additional benefit? The only case I can think of is if the method were to be tested separately outside of the apply.

Reply:
I don't believe it adds anything. I added it to preserve the behavior but I didn't piece together it was private. I'll clean this up.